builder/builder-next/builder.go

```diff
@@ -221,7 +221,9 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 
 	id := identity.NewID()
 
-	frontendAttrs := map[string]string{}
+	frontendAttrs := map[string]string{
+		"override-copy-image": "docker.io/docker/dockerfile-copy:v0.1.7@sha256:4211460d4df58cca572825b93e437c09dac6387d88910fe07ac8f7c78d52e0ce",
```

Comment:
the upstream repos (moby/moby and moby/buildkit both have version v0.1.5 committed, not v0.1.7. Also where is this built from?

Reply:
https://github.com/tonistiigi/copy/releases/tag/v0.1.7

Reply:
No multiarch support?

Reply:
> No multiarch support?

```
» docker run --rm mplatform/mquery docker.io/docker/dockerfile-copy:v0.1.7@sha256:4211460d4df58cca572825b93e437c09dac6387d88910fe07ac8f7c78d52e0ce
Image: docker.io/docker/dockerfile-copy:v0.1.7@sha256:4211460d4df58cca572825b93e437c09dac6387d88910fe07ac8f7c78d52e0ce
 * Manifest List: Yes
 * Supported platforms:
   - linux/amd64
   - linux/arm/v6
   - linux/arm/v7
   - linux/arm64/undefined
   - linux/s390x
   - linux/ppc64le
```

> moby/buildkit both have version v0.1.5 committed,

https://github.com/moby/buildkit/pull/669

Reply:
ah cool!